package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "mock-fs",
   "description": "A configurable mock file system.  You know, for testing.",
-  "version": "3.6.0",
+  "version": "3.6.1",
```

Comment:
Please leave the version number unchanged.  I'll bump it before releasing (a release may include changes from multiple feature branches).